Doc/library/http.server.rst

```diff
@@ -462,6 +497,17 @@ following command runs an HTTP/1.1 conformant server::
 .. versionchanged:: 3.11
    Added the ``--protocol`` option.
 
+The server can also support TLS encryption. The options ``--tls-cert`` and
+``--tls-key`` allow specifying a TLS certificate chain and private key for
+secure HTTPS connections. Use ``--tls-password`` option if private keys are
```

Comment:
Passwords should never be put on a command line.  Don't add that flag unless it is named `--my-tls-password-is-not-secret-anymore-because-I-passed-it-on-a-command-line=` to tell people never to use it.

An appropriate way to get a password is from a file.  `--tls-password-file=` would be acceptable.

If you want to support use of `getpass`, make an explicit flag for just that.